tests/lib/ecma-features.js

```diff
@@ -57,8 +56,7 @@ describe("ecmaFeatures", () => {
             ecmaFeatures: {}
         };
     });
-
-    leche.withData(testFiles, filename => {
+    testFiles.forEach(filename => {
```

Comment:
We actually need a `describe` block in here with the filename. That’s part of what leche does. Otherwise, we can’t tell which file caused a failure.

Reply:
I see! will add a test-util to handle this.